workspaces/global-header/plugins/global-header/src/components/HeaderDropdownComponent/StarredDropdown.tsx

```diff
@@ -70,6 +73,13 @@ const StarredItem: FC<SectionComponentProps> = ({
       onClick={handleClose}
       disableRipple
       disableTouchRipple
+      onMouseEnter={() => setIsHovered(true)}
+      onMouseLeave={() => setIsHovered(false)}
+      sx={{
+        '&:hover .star-icon': {
+          visibility: 'visible',
+        },
+      }}
```

Comment:
Didn't you try to archive the same, once with a css rule and once with a state? Doesn't this work just with a css rule maybe?